src/components/Vulnurabilities/CcviThermometer/CcviThermometer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { CcviLevel, getCcviLevelColor } from 'common/ccvi';
+import { scaleLinear } from '@vx/scale';
+import { v4 as uuidv4 } from 'uuid';
+
+const CcviThermometer: React.FC<{ overallScore: number }> = ({
+  overallScore,
+}) => {
+  const gradientId = uuidv4();
+  const containerHeight = 34;
+  const thermometerWidth = 240;
+  const thermometerHeight = 20;
+
+  const scaleWidth = scaleLinear({
+    domain: [0, 1],
+    range: [0, thermometerWidth],
+  });
+
+  const pointerX = scaleWidth(overallScore);
+
+  return (
+    <svg width={thermometerWidth} height={containerHeight}>
+      <defs>
+        <linearGradient id={gradientId}>
+          <stop
+            offset="20%"
+            stop-color={getCcviLevelColor(CcviLevel.VERY_LOW)}
+          />
+          <stop offset="20%" stop-color={getCcviLevelColor(CcviLevel.LOW)} />
+          <stop offset="40%" stop-color={getCcviLevelColor(CcviLevel.LOW)} />
+          <stop offset="40%" stop-color={getCcviLevelColor(CcviLevel.MEDIUM)} />
+          <stop offset="60%" stop-color={getCcviLevelColor(CcviLevel.MEDIUM)} />
+          <stop offset="60%" stop-color={getCcviLevelColor(CcviLevel.HIGH)} />
+          <stop offset="80%" stop-color={getCcviLevelColor(CcviLevel.HIGH)} />
+          <stop
+            offset="80%"
+            stop-color={getCcviLevelColor(CcviLevel.VERY_HIGH)}
+          />
```

Comment:
In react components, we need to use `stopColor` instead of `stop-color` or else we get warnings in the console (still works though)